From 197d3dc3b9acb93be30c4a7f49e8c0ad3f554818 Mon Sep 17 00:00:00 2001
From: Jan Blechta <jan.blechta@math.tu-chemnitz.de>
Date: Wed, 28 Apr 2021 18:57:47 +0200
Subject: [PATCH] Fix regression in build system

Reverts small part of 99832622e157067e40896815ace0f8f7a153e8af.
EFI_APP_LOCATION conf var (or FWUPD_EFIAPPDIR env var) is needed for

  fu_common_get_path (FU_KIND_PATH_EFIAPPDIR)

to function properly
---
 meson.build | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/meson.build b/meson.build
index 398d76ab..2e7fbd02 100644
--- a/meson.build
+++ b/meson.build
@@ -369,6 +369,9 @@ endif
 if build_standalone and get_option('plugin_uefi_capsule')
   efiboot = dependency('efiboot')
 
+  efi_app_location = join_paths(libexecdir, 'fwupd', 'efi')
+  conf.set_quoted('EFI_APP_LOCATION', efi_app_location)
+
   if host_cpu == 'x86'
           EFI_MACHINE_TYPE_NAME = 'ia32'
   elif host_cpu == 'x86_64'
-- 
2.31.1

